anaconda.spec.in

```diff
@@ -145,6 +146,10 @@ Requires: newt-python
 # required because of the rescue mode and VNC question
 Requires: anaconda-tui = %{version}-%{release}
 
+# Storage packages that blivet considers optional
+Requires: libblockdev-plugins-all >= %{libblockdevver}
```

Comment:
The idea behind this is that the ``libblockdev`` package will still be required by Blivet, right? I think we need some comment about it in the spec.